src/Microsoft.Framework.Runtime.Hosting/RuntimeLogging.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Runtime.Hosting.Internal;
+
+namespace Microsoft.Framework.Runtime
+{
+    public static class RuntimeLogging
+    {
+        private static ILoggerFactory _loggerFactory;
+        private static Func<string, LogLevel, bool> _filter;
+
+        public static Func<string, LogLevel, bool> Filter { get { return _filter; } }
+        public static bool IsEnabled { get { return _filter != null; } }
+
+        public static void Initialize(Func<ILoggerFactory> loggerFactoryFactory)
```

Comment:
Yep, I went there. Logger Factory Factory :). I don't want to run the delegate if the KRE_TRACE variable is empty, so that's what you get!